mergify_engine/actions/merge/helpers.py

```diff
@@ -73,6 +83,21 @@ def merge_report(ctxt, strict):
     return conclusion, title, summary
 
 
+def get_queue_summary(q):
+    pulls = q.get_pulls()
+    if not pulls:
+        return ""
+
+    summary = "\n\nThe following pull requests are queued:"
```

Comment:
nitpick, but I'd put the first `\n\n` on L109, not here.

Reply:
I wanted to avoid an `if` when this function end earlier, other we will have 4 `\n`

Reply:
fair enough